documentation/slate/source/includes/apis.md

```diff
@@ -878,6 +878,43 @@ direction       | int                    | Angle (in degree) between the previou
 <a name="isochrones_api"></a>Isochrones (currently in Beta)
 ---------------------------------------
 
+``` shell
+curl 'https://api.navitia.io/v1/coverage/sandbox/isochrones?from=stop_area:RAT:SA:GDLYO&max_duration=3600' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+
+HTTP/1.1 200 OK
```

Comment:
You should add "#Request" and "#Response" comments, as it is done in the other example:
`#Request`
`$ curl 'https://api.navitia.io/v1/coverage/sandbox/...' `

`#Response`
`HTTP/1.1 200 OK`